packages/nxengine/build.sh

```diff
@@ -0,0 +1,18 @@
+TERMUX_PKG_HOMEPAGE=https://nxengine.sourceforge.net
+TERMUX_PKG_DESCRIPTION="Open-source rewrite engine of the Cave Story for Dingux and MotoMAGX. Author - Caitlin Shaw (rogueeve), httpd://nxengine.sourceforge.net"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@Yonle"
+TERMUX_PKG_VERSION=1.0.0.4
+TERMUX_PKG_REVISION=4
```

Comment:
Why revision starts from 4? This is a new package and therefore it shouldn't have revision field set. Revision is a build number and incremented each time when package should be rebuilt.